src/components/Text/src/Text.vue

```diff
@@ -26,7 +26,7 @@ export default {
 		element: {
 			type: String,
 			default: 'p',
-			validator: (element) => ['p', 'span', 'div', 'li'].includes(element),
+			validator: (element) => ['p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'span', 'div', 'li'].includes(element),
```

Comment:
adding this makes the mheading component irrelevant/useless. it should be removed.

Reply:
It does, I agree. I figured the follow-up work from this PR would be the breaking changes we're planning.